plan_runner/open_left_door.py

```diff
@@ -39,6 +44,28 @@ def InverseKinPointwise(p_WQ_start, p_WQ_end, duration,
                         position_tolerance=0.005,
                         theta_bound=0.005 * np.pi, # 0.9 degrees
                         is_printing=True):
+    """
+    Calculates a joint space trajectory for iiwa by repeatedly calling IK. The first IK is initialized (seeded)
+    with q_initial_guess. Subsequent IKs are seeded with the solution from the previous IK.
+
+    Positions for point Q (p_EQ) and orientations for the end effector, generated respectively by
+    InterpolatePosition and InterpolateOrientation, are added to the IKs as constraints.
+
+    :param p_WQ_start: The first argument of function InterpolatePosition (defined below).
```

Comment:
data types and sizes (eg. [3 x 1] numpy array) would be helpful for all of these parameters

Reply:
They don't have to be (3,) numpy arrays. They are inputs to the `InterpolatePosition` function defined later, and their data type depend on the definition of the function.